src/app/page.tsx

```diff
@@ -1,5 +1,490 @@
+            <div className="flex flex-col gap-[4px] lg:gap-[22px]">
+              <p className="text-center text-[20px] leading-[27px] font-[700] text-black">££££</p>
+              <div className="flex flex-col">
+                <div className="lg:py-[30px] h-[177px] lg:h-[326px] text-center bg-yellow text-[16px] lg:text-[20px] leading-[18px] lg:leading-[23px] font-[700] flex justify-center items-center w-full px-[10px] lg:px-[30px] text-center">Typical Advertising Spend</div>
+                <div className="px-[5px] py-[25px] lg:py-[30px] text-center lg:h-[82px] bg-yello1 text-[15px] lg:text-[20px] leading-[17px] lg:leading-[23px] font-[700] flex justify-center items-center w-full">The Middlemen’s Cut</div>
+                <div className="px-[5px] py-[14px] lg:py-[30px] text-center bg-white text-[12px] lg:text-[20px] leading-[14px] lg:leading-[23px] font-[700] flex justify-center items-center w-full">The Cost of Ingredients</div>
+              </div>
+              <p className="text-center text-[12px] lg:text-[18px] leading-[14px] lg:leading-[20px] font-[700] text-white">What brands charge</p>
+            </div>
+            <div className="flex flex-col gap-[4px] lg:gap-[22px]">
+              <p className="text-center text-[20px] leading-[27px] font-[700] text-black">£</p>
+              <Button className="bg-blue text-white w-full font-bold h-[58px] lg:h-[107px] text-[18px] lg:text-[24px] leading-[18px] lg:leading-[27px]" asChild>
+                <Link href="/about">Club Price</Link>
+              </Button>
+              <p className="text-center text-[12px] lg:text-[18px] leading-[14px] lg:leading-[20px] font-[700] text-white">What we charge</p>
+            </div>
+          </div>
+        </div>
+
+        <div className="flex flex-col justify-between">
+          <div>
+            <p className="text-[24px] lg:text-[40px] leading-[32px] lg:leading-[58px] font-[700] text-blue pt-[32px] lg:pt-[18px] pb-[16px]">Pharmaceutical grade Ingredients aren’t expensive. Marketing them is.</p>
+            <p className="text-[14px] leading-[26px] text-grey6">Before Supplement Club, you were forced to pay for bloated advertising costs and watered-down supplements. We cut out the advertising overheads and middlemen and send you 100% clinically effective, traceable ingredients.
+              <br />
+              <br />
+              With Supplement Club, you get direct access to top labs in Japan, the US, and Europe. Our transparent model shows you exactly where your supplements come from—no more blindly trusting brands. You can research yourself and know you’re getting the best money can buy with 100% transparency.
+              <br />
+              <br />
+              <br />
+            </p>
+            <p className="font-[700] mb-[32px] lg:mb-[18px]">Supplement Club Subscribers don't pay for:</p>
```

Comment:
Eslint is complaining about `don't` here:

Can we escape it instead like this:

`don&apos;t`